man/man8/zpool.8

```diff
@@ -26,8 +26,8 @@
 .\" Copyright (c) 2017 George Melikov. All Rights Reserved.
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\"
-.Dd June 28, 2017
-.Dt ZPOOL 8 SMM
+.Dd August 23, 2017
+.Dt ZPOOL 1M
```

Comment:
We do not want to change this back to `1M`, it need to remain `8 SMM` for Linux.